### PR TITLE
crdroid: QS Header Image [2/2]

### PR DESCRIPTION
**Pull request by awakened1712**

### CODE REVIEW

**Comment by ShevT** [comment]
[Here](https://github.com/crdroidandroid/android_packages_apps_crDroidSettings/pull/1049/commits/90c6c649a8d64b24535466d3ea2732c09b122c43#diff-48166cc0f4035b1423e5854b658f678ab6311279e64ce7a382d89256682c5668R127) you need to replace tabs with spaces

**Comment by ShevT** [comment]
@gwolf2u @neobuddy89 Personally, I like it. It looks cool.

**Comment by neobuddy89** [comment]
why are there unused strings?

File: res/values/cr_strings.xml [hunk 1]
```diff
@@ -1261,4 +1261,9 @@
 
     <!-- Radio Info -->
     <string name="radio_info_title">Radio info</string>
+
+    <!-- QS Header -->
+    <string name="qs_header_image_title">QS Header Image</string>
+    <string name="qs_header_random_image_title">QS Header Random Image</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml]: Not used

File: res/values/cr_strings.xml [hunk 2]
```diff
@@ -1261,4 +1261,9 @@
...
+    <string name="qs_header_random_image_summary">Enable random QS header image</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml]: Neither this one

**Review by gwolf2u — CHANGES_REQUESTED** [review]
qs_header_random_image_summary and qs_header_random_image_title are not used anywhere indeed

**Pull request closed without merging**